CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `com.google.protobuf:protobuf-java` from 3.22.0 to 3.22.2
 - Bump Netty to 4.1.90.Final ([#6677](https://github.com/opensearch-project/OpenSearch/pull/6677)
 - Bump `com.diffplug.spotless` from 6.15.0 to 6.17.0
+- Bump `org.apache.zookeeper:zookeeper` from 3.8.0 to 3.8.1
```

Comment:
That seems to be taking changes from other backports?

Reply:
I added this to avoid conflicts and pushing again. That's why added in the PR description to merge this PR after #6768 is merged.

Reply:
Got the conflict eventually. Updated it